### PR TITLE
Protocol explore: add liquidation form with gas token switch

### PR DESCRIPTION
**Pull request by OlehX**
https://app.clubhouse.io/bzx1/story/879/add-liquidation-form

### CODE REVIEW

**Comment by OlehX** [comment]
https://app.clubhouse.io/bzx1/story/879/add-liquidation-form

**Comment by Kharabet** [comment]
Shouldn't be merged until profit calculation isn't ready

**Comment by AoDev** [comment]
Please rebase this on latest development branch first

**Comment by OlehX** [comment]
@AoDev done

**Review by AoDev — APPROVED** [review]
Looks good. There are some minor details I commented.

File: packages/protocol-explorer/src/pages/LiquidationsPage.tsx [hunk 1]
```diff
@@ -98,47 +102,47 @@ export class LiquidationsPage extends Component<ILiquidationsPageProps, ILiquida
...
+      return xs.reduce((rv: any, x: any) => {
         ;(rv[x[key]] = rv[x[key]] || []).push(x)
         return rv
       }, {})
     }
     const eventsWithDay = events.map(
       (e: { event: LiquidationEvent; repayAmountUsd: BigNumber }) => ({
         ...e,
-        day: parseInt((e.event.timeStamp.getTime() / (1000 * 60 * 60 * 24)).toString())
+        day: parseInt((e.event.timeStamp.getTime() / (1000 * 60 * 60 * 24)).toString(), undefined)
```
[the remark this answers is not in the document]
Comment (Kharabet) [reply packages/protocol-explorer/src/pages/LiquidationsPage.tsx]: probably we have to use it something like `Math.floor`

**Pull request merged**